src/Handlers/UpdateHandler.php

```diff
@@ -86,6 +86,6 @@ public function __invoke($next)
             $result = $this->handle();
         }
 
-        return $result ?? $next();
+        return $result ?? (is_callable($next) ? $next() : null);
```

Comment:
There is no such case actually when the `$next = null`. The runner should always yield and return callable, so I wonder how you end up having this issue: https://github.com/westacks/telebot/blob/363fc68d4fc6c8c60be98e6ce61ef99448582037/src/Kernel.php#L69-L80

Please provide how you end up with it, and I'll accept it gladly if this condition is necessary.